helm/thingsboard/templates/node.yaml

```diff
@@ -64,7 +64,7 @@ spec:
         - name: check-db-queue-ready
           image: {{ include "thingsboard.postgresImage" . }}
           command: ['sh', '-c',
-            'export PGPASSWORD=''{{ index .Values "postgresql-ha" "postgresql" "password" }}'' && export row_count=0 && until [ $row_count -ge 3 ]; do sleep 1; row_count=$(psql -h {{ include "thingsboard.pgpoolservicename" . }} -p {{ index .Values "postgresql-ha" "pgpool" "containerPorts" "postgresql" }} -U {{ index .Values "postgresql-ha" "postgresql" "username" }} -d {{ index .Values "postgresql-ha" "postgresql" "database" }} --set=on_error_stop=1 -t -c "SELECT count(*) FROM queue;"); done;']
+            'export PGPASSWORD=''{{ index .Values "postgresql-ha" "postgresql" "password" }}'' && export row_count=0 && until [ $row_count -ge 3 ]; do sleep 1; row_count=$(psql -h {{ include "thingsboard.pgpoolservicename" . }} -p {{ index .Values "postgresql-ha" "pgpool" "containerPorts" "postgresql" }} -U {{ index .Values "postgresql-ha" "postgresql" "username" }} -d {{ index .Values "postgresql-ha" "postgresql" "database" }} --set=on_error_stop=1 -t -c "SELECT count(*) FROM queue WHERE partitions={{ .Values.postgresInitDB.job.check.partitions }};"); done;']
```

Comment:
why do we really need the where clause? I guess that the wait should be enough for having the queues created. WDYT?

Reply:
The 'where' is needed to avoid race in tb-node start. The db_init install script creates tables with 'partition' column set to 10. Then, in the next line we modify this table to decrease the number of partitions. So, it might happen that tb-node starts with default value of 10. I don't know if/when this value will be updated for the tb-node, we might need to restart the service for that. Does it make sense?